.github/workflows/integration.yml

```diff
@@ -14,37 +14,87 @@ permissions:
   contents: read
 
 jobs:
-  critest:
-    runs-on: ubuntu-latest
+  test-binaries:
+    strategy:
+      matrix:
+        run:
+          - runner: ubuntu-latest
+            arch: amd64
+          - runner: actuated-arm64-4cpu-16gb
```

Comment:
Do you want to add in metering to see what kind of usage it tends to run up?

https://gist.github.com/alexellis/1f33e581c75e11e161fe613c46180771

Reply:
Yep, done :+1: